app/src/main/java/com/buzzshelter/Controllers/CSVFile.java

```diff
@@ -0,0 +1,46 @@
+package com.buzzshelter.Controllers;
+
+/**
+ * Created by jeffr on 2/20/2018.
+ */
+
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CSVFile {
+    InputStream inputStream;
```

Comment:
I would make this private

Reply:
When I try to make it private, Android Studio gives me a warning that says that "private is not allowed here."